lib/sender.js

```diff
@@ -29,18 +29,27 @@ var parseAndRespond = function(resBody, callback) {
 function extractRecipient(recipient) {
     var validKeys = ['registrationIds', 'registrationTokens', 'topic', 'notificationKey'];
     var theRecipient;
-
-    validKeys.forEach(function(key) {
-        if (recipient[key]) {
+    var err = null;
+    var key = null;
```

Comment:
There is no reason to instantiate `key` here. It is never used outside the loop.